boundary_layer_default_plugin/config/operators/dataproc_cluster_delete.yaml

```diff
@@ -20,15 +20,14 @@ parameters_jsonschema:
     properties:
         cluster_name:
             type: string
+        project_id:
+            type: string
         region:
             type: string
-        priority_weight:
```

Comment:
why delete this?  This was an intentional override of the default setting, to prioritize cluster deletion above (hopefully) all other operations.  We wouldn't want a cluster that's eligible for deletion to sit around waiting on other ordinary tasks, in case the dag's concurrency limit were reached.